### ISSUE TITLE
Building with no symbol versioning is broken

### ISSUE THREAD

**Issue by galaxy4public** [besser82/libxcrypt#142]
TL;DR -- `./configure --disable-symvers` produces a configuration one cannot actually build since it will fail during the "make" phase.

Although `./configure` provides an option to disable symbol versioning (i.e. `--disable-symvers`) it does not really work since there are multiple places in the source code where symbol versioning is engaged.  For starters, the `./build-aux/scripts/gen-crypt-symbol-vers-h` script produces a header with versioning info.  I am building this library on a musl-based system and musl does not support versioning (well, it can work with it, but I try to keep it clean, so don't want to have any symvers crawling into the libraries).  Perhaps, I will submit a patch for this issue soon.
